reference/5.0/PowershellGet/Find-Module.md

```diff
@@ -15,66 +15,83 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
```

Comment:
Change the parameter set name to "All".

Reply:
Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.